packages/clay-core/src/index.ts

```diff
@@ -19,3 +19,5 @@ export {
 	useModal,
 } from '@clayui/modal';
 export {Provider, useProvider} from '@clayui/provider';
+
+export {TreeView as UNSAFE_TreeView} from './tree-view';
```

Comment:
Just a question, why are we exporting this with `UNSAFE_`, it's not that I don't agree, but I thought we've done a pretty good job up to now with this component, so I'd like to understand.

Reply:
Yeah, just adding it as "unstable" to at least have it handy and we can go ahead with the implementations on the DXP side until we're sure nothing's missing, let's work on the next epic of removing TreeView usage from DXP in exchange for Clay, we can remove the flag in the next version anyway.

Reply:
Fair enough, OK ... that said 

> let's work on the next epic of removing TreeView usage from DXP in exchange for Clay, we can remove the flag in the next version anyway.

I'd say: let's help replacing the old TreeView with the new one: if we do it all by ourselves, we're not really helping the team understanding how to use our component. I know we might have to iterate between DXP and Clay to add/fix things that are missing, but I think it's better to involve the team (at least one of their members) on this journey :smile:

Reply:
haha yeah, I agree.